### PR TITLE
FND-280 #comment - loosens constraints on business day related proper…

### PR DESCRIPTION
**Pull request by ElisaKendall**
…ties to eliminate a reasoning bug, eliminates a duplicate individual that was not properly defined, and normalizes definitions to be ISO 704 compliant

Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

This issue resolution loosens constraints on business day related properties to eliminate a reasoning bug, eliminates a duplicate individual that was not properly defined, and normalizes definitions to be ISO 704 compliant.

Fixes: #895 / FND-280

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Comment by ElisaKendall** [comment]
Yes, there is the possibility of adding unions to constrain this better, but I'd rather pause on that until I've had more time to work with the examples and see what makes sense.  What Mark had originally didn't work as he intended, though, and we ended up with logical inconsistencies in examples as a consequence.

I'll continue to work through this and then possibly raise another issue to further constrain their use.  Thanks!

File: FND/DatesAndTimes/BusinessDates.rdf [hunk 1]
```diff
@@ -75,54 +76,54 @@ They are modularized this way to minimize the ontological committments that are
...
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessDayNearest">
 		<rdf:type rdf:resource="&fibo-fnd-dt-bd;BusinessDayConvention"/>
 		<rdfs:label>business day nearest</rdfs:label>
 		<sm:normativeReference>FPML 5.1 &quot;BusinessDayConventionEnum&quot;</sm:normativeReference>
-		<skos:definition>BusinessDayNearest is a BusinessDayConvention that means that a non-business date will be adjusted to the nearest day that is a business day -- i.e. if the non-business day falls on any day other than a Sunday or a Monday, it will be the first preceding day that is a business day, and will be the first following business day if it falls on a Sunday or a Monday.</skos:definition>
+		<skos:definition>convention specifying that a non-business date will be adjusted to the nearest day that is a business day -- i.e. if the non-business day falls on any day other than a Sunday or a Monday, it will be the first preceding day that is a business day, and will be the first following business day if it falls on a Sunday or a Monday</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/BusinessDates.rdf]: "first preceding" could be taken as "earliest preceding". "latest preceding would be clearer".
The algorithm is unnecessarily specific and wrong - in the case of the Saturday after Thanksgiving (assuming the Friday is not a business day) then the nearest business day will be the following Monday but the algorithm says the preceding Wednesday.

File: FND/DatesAndTimes/BusinessDates.rdf [hunk 2]
```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
...
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalConvention">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-bd;Convention"/>
 		<rdfs:label>business recurrence interval convention</rdfs:label>
-		<skos:definition>a convention for modeling recurring days, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
+		<skos:definition>convention that specifies how recurring days should be handled, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
 	</owl:Class>
 	
-	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalEndOfMonth">
```
Comment (rivettp) [inline FND/DatesAndTimes/BusinessDates.rdf]: Should this not be deprecated and made equivalentTo EndOfMonth?

File: FND/DatesAndTimes/BusinessDates.rdf [hunk 3]
```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business recurrence interval</rdfs:label>
-		<skos:definition>a recurrence interval that is specified using a business recurrence interval convention</skos:definition>
+		<skos:definition>recurrence interval that is specified using a business recurrence interval convention</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/BusinessDates.rdf]: Outside scope of this PR, but this is a terrible class name for this definition.

File: FND/DatesAndTimes/BusinessDates.rdf [hunk 4]
```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
...
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalConvention">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-bd;Convention"/>
 		<rdfs:label>business recurrence interval convention</rdfs:label>
-		<skos:definition>a convention for modeling recurring days, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
+		<skos:definition>convention that specifies how recurring days should be handled, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/BusinessDates.rdf]: I don't see why t-bill auction is here, and it's very jurisdiction-specific

File: FND/DatesAndTimes/BusinessDates.rdf (hunk 2 above)
Comment (ElisaKendall) [reply FND/DatesAndTimes/BusinessDates.rdf]: We had agreed that for FIBO 2.0 we were not deprecating anything until after the FTF work is done.

File: FND/DatesAndTimes/BusinessDates.rdf (hunk 4 above)
Comment (ElisaKendall) [reply FND/DatesAndTimes/BusinessDates.rdf]: It was there before - all I did was fix the definition.  And there are t-bills referenced in the FpML that may not be limited to the US, including Australian t-bills ... although those could be Australian dollar but US t-bills.

File: FND/DatesAndTimes/BusinessDates.rdf (hunk 1 above)
Comment (ElisaKendall) [reply FND/DatesAndTimes/BusinessDates.rdf]: Let's address this under a new issue - I only normalized the definition, did not change the content.

File: FND/DatesAndTimes/BusinessDates.rdf (hunk 3 above)
Comment (rivettp) [reply FND/DatesAndTimes/BusinessDates.rdf]: Created #899

**Pull request merged**